Sources/XcodesKit/XcodeInstaller.swift

```diff
@@ -162,7 +162,7 @@ public final class XcodeInstaller {
                 guard attemptNumber < 1 else { throw error }
 
                 switch installationType {
-                case .url:
+                case .path:
                     // If the user provided the URL, don't try to recover and leave it up to them.
```

Comment:
```suggestion
                    // If the user provided the path, don't try to recover and leave it up to them.
```
One tiny last thing 😄

Reply:
Done